# Week 7: PX4/ArduPilot & ROS2 Integration Parameters

px4_telemetry_monitor:
  ros__parameters:
    update_rate_hz: 2.0

offboard_commander:
  ros__parameters:
    waypoints:
      - "[0.0, 0.0, -5.0]"
      - "[10.0, 0.0, -5.0]"
      - "[10.0, 10.0, -5.0]"
      - "[0.0, 10.0, -5.0]"
      - "[0.0, 0.0, -5.0]"
    waypoint_radius: 1.0
    cruise_speed: 2.0

geofence_monitor:
  ros__parameters:
    fence_center_x: 0.0
    fence_center_y: 0.0
    fence_radius: 50.0
    fence_max_height: 30.0
